### PR TITLE
Add a scrolling test now that it passes

### PR DESCRIPTION
**Pull request by abarth**
We fixed this issue a different way but the test is still valuable.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/test/widget/scroll_events_test.dart [hunk 1]
```diff
@@ -136,4 +136,21 @@ void main() {
...
+      List<String> log = <String>[];
+      tester.pumpWidget(_buildScroller(key: scrollKey, log: log));
+
+      expect(log, equals([]));
+      tester.flingFrom(new Point(100.0, 100.0), new Offset(50.0, 50.0), 500.0);
+      tester.pump(new Duration(seconds: 1));
+      tester.pump(new Duration(seconds: 1));
+      tester.pump(new Duration(seconds: 1));
+      log.removeWhere((String value) => value == 'scroll');
+      expect(log, equals(['scrollstart', 'scrollend']));
```
Comment (Hixie) [inline packages/flutter/test/widget/scroll_events_test.dart]: `<String>`

Comment (Hixie) [inline packages/flutter/test/widget/scroll_events_test.dart]: could also just check that the first is a scroll, the last is an end, and that nothing is left if you remove the scrolls.

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/test/widget/scroll_events_test.dart (hunk 1 above)
Comment (abarth) [inline packages/flutter/test/widget/scroll_events_test.dart]: Yeah, ok.  That's a better test.

**Pull request merged**